docs/01-app/03-api-reference/03-file-conventions/middleware.mdx

```diff
@@ -669,6 +669,75 @@ expect(getRewrittenUrl(response)).toEqual('https://other-domain.com/docs')
 
 Learn how to [configure Middleware](/docs/app/guides/self-hosting#middleware) when self-hosting Next.js.
 
+## Troubleshooting
+
+### Images not loading with authentication middleware
+
+If images fail to load on authenticated routes, ensure `_next/image` is excluded from the matcher:
```

Comment:
As said above, this is not caused by `_next/image` in middleware, rather, the Image Optimizer after the middleware pass, doesn't have credentials, so if it requests internal images also protected by the middleware, these won't be shown